packages/labs/motion/src/view-transition-update.ts

```diff
@@ -0,0 +1,48 @@
+import {LitElement} from 'lit';
```

Comment:
Add license header like we have in other files, copyright 2023

Reply:
Done e62776a9dd534c451d6530eeb4c21a347f842f30